carapp/authent.py

```diff
@@ -0,0 +1,20 @@
+from carapp import auth, db_models, login_manager, token_serializer
+
+
+@login_manager.user_loader
+def user_loader(user_id):
+    user = db_models.User.query.get(user_id)
+    if not user:
+        return None
+
+    return user
+
+
+@auth.verify_token
+def verify_token(token):
+    try:
+        data = token_serializer.loads(token)
+    except:
+        return False
+    if "username" in data:
+        return data["username"]
```

Comment:
```suggestion
    data = token_serializer.loads(token)
    return data.get("username", False)
```

Same thing, too many moves.

Reply:
Wow, this one is a nice move 💯 
I try to solve katas like this all the time on codewars but missed it here.

Reply:
It is just a habit. What you've done yourself is totally valid too.